scripts/dataplan_package/template/README.md

```diff
@@ -0,0 +1,81 @@
+# {{ package_pprint_name }} client library for Python
+<!-- write necessary description of service -->
+
+## Getting started
+
+### Installating the package
+
+```bash
+python -m pip install {{ package_name }}
+```
+
+#### Prequisites
+
+- Python 2.7, or 3.6 or later is required to use this package.
+- You need an [Azure subscription][azure_sub] to use this package.
```

Comment:
Do we need to specify both [Azure subscription] and [azure_sub] here? I didn't see any replacement of those values.

Reply:
updated the doc link:
```
[azure_sub]: https://azure.microsoft.com/free/
```